dipy/segment/tests/test_clustering.py

```diff
@@ -356,14 +369,16 @@ def test_cluster_map_iter():
         cluster_map.add_cluster(new_cluster)
         clusters.append(new_cluster)
 
-    assert_true(all([c1 is c2 for c1, c2 in zip(cluster_map.clusters, clusters)]))
+    assert_true(
+        all([c1 is c2 for c1, c2 in zip(cluster_map.clusters, clusters)]))
     assert_array_equal(cluster_map, clusters)
     assert_array_equal(cluster_map.clusters, clusters)
     assert_array_equal(cluster_map, [cluster.indices for cluster in clusters])
 
     # Set refdata
     cluster_map.refdata = data
-    assert_array_equal(cluster_map, [[data[i] for i in cluster.indices] for cluster in clusters])
+    assert_array_equal(
+        cluster_map, [[data[i] for i in cluster.indices] for cluster in clusters])
```

Comment:
It would be better as:
```python
assert_array_equal(cluster_map,
                   [[data[i] for i in cluster.indices] for cluster in clusters])
```